data/sources/00_areatree

```diff
@@ -114,7 +114,7 @@
     1717:Ismaninger Str. 33, Hörzentrum (Bau 717):
     -1718:Bau 718:
     -1719:Bau 719:
-    1720:Grillparzerstraße 16-18, Institut für Medizinische Statistik und Epidemiologie (IMSE):mri-imse[building]
+    1720:Grillparzerstraße 16-18, Institut für Medizinische Statistik und Epidemiologie (IMSE):1720[building]
```

Comment:
Yes. The reason is that's been the only building where the `id` is different from the `b_prefix` (because it's a building not from TUMonline). This greatly speeds up the code to find the internal id of a building